app/views/content/shared/age-groups-and-specialisms/_check_qualifcations.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="check-qualifcations">
```

Comment:
Actually - and far more importantly - this is almost an exact duplicate of https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/views/content/shared/qualifications-training/_check_qualifcations.html.erb which is the partial used on the subject pages

The only difference I can see is:
* this one - `Having relevant A levels can show your subject knowledge, if you do not have a degree in the subject you want to teach.`
* subjects one - `Having relevant A levels can show your subject knowledge, if you do not have a degree in <%= @front_matter["subject"] %>.`

I don't think the benefit of the slightly different wording outweighs having two partials to maintain?

I would suggest that rather than creating a new partial, you update the one used on the subject pages to slightly more generic ie `subject you want to teach`. I think there is still enough mention of the subject in the Quals section on the subject pages for this not to make too much difference?

But do discuss with @Sarah-DfE and disagree if needed!

(Either way, this partial should not be in `age-groups-and specialisms` - the folder should relate to what the partial is about, not where it is used!)